java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java

```diff
@@ -0,0 +1,61 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.GeoAddOptions;
+import glide.api.models.commands.GeospatialData;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Geospatial Indices Commands" group for standalone and
+ * cluster clients.
+ *
+ * @see <a href="https://redis.io/commands/?group=geo">Geospatial Indices Commands</a>
+ */
+public interface GeospatialIndicesBaseCommands {
```

Comment:
Should we keep such long name here too?
```suggestion
public interface GeospatialBaseCommands {
```
```suggestion
public interface GeoDataBaseCommands {
```

Reply:
This is the official name of the group from Redis don't see harm in keeping it.